Formula/bazel.rb

```diff
@@ -15,6 +16,9 @@ class Bazel < Formula
 
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
+    # Bazel has it owns sandbox, ensure it works.
+    ENV.delete("SDKROOT")
+    ENV["CC"] = `xcrun -f #{ENV["CC"]}`.strip
```

Comment:
This is a pretty invasive change and almost completely works around our `superenv` used for sanitizing compiler flags. I'd prefer if this can be avoided. Can you explain in more detail why you can't just leave `CC` unset or make it `CC=cc`? What issues would that create for Bazel's build system?